.github/workflows/backport-pr.yml

```diff
@@ -0,0 +1,31 @@
+name: Backport PR
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
```

Comment:
```suggestion

```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Discussed offline: seems too error prone for the label to be triggered on `labeled`. One misclick and your PR gets backported